test/helpers/docker.go

```diff
@@ -92,6 +92,23 @@ func (s *SSHMeta) ContainerInspectNet(name string) (map[string]string, error) {
 	return result, nil
 }
 
+// ContainerInspectIP returns the IPv4 and IPv6 strings for the container with
+// the provided name
+func (s *SSHMeta) ContainerInspectIP(name string) (string, string, error) {
```

Comment:
Why you added a new helper? This is already in place with `vm.ContainerInspectNet()` no?

Reply:
Just a bit easier to use IMO returns a tuple vs a map but I have no problem pushing it over to InspectNet.

Reply:
Although looking at it now I only use it in a couple spots so really doesn't make much sense. Using InspectNet() now.